app/services/call_creator.rb

```diff
@@ -92,6 +96,11 @@ def place_call
     end
   end
 
+  def publish_event
+    return if @call.member.blank? # handle this in worker
```

Comment:
what does this do? and why not put this conditional with the other two (`@call.persisted?` and `!@call.failed?`)